[2024-09-10T07:35:35.026+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-10T07:35:35.092+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_V3.third_task scheduled__2024-04-15T00:00:00+00:00 [queued]>
[2024-09-10T07:35:35.128+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_V3.third_task scheduled__2024-04-15T00:00:00+00:00 [queued]>
[2024-09-10T07:35:35.131+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-09-10T07:35:35.222+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): third_task> on 2024-04-15 00:00:00+00:00
[2024-09-10T07:35:35.235+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=23725) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-10T07:35:35.240+0000] {standard_task_runner.py:72} INFO - Started process 23738 to run task
[2024-09-10T07:35:35.235+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_V3', 'third_task', 'scheduled__2024-04-15T00:00:00+00:00', '--job-id', '5431', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpx1cg_bk0']
[2024-09-10T07:35:35.260+0000] {standard_task_runner.py:105} INFO - Job 5431: Subtask third_task
[2024-09-10T07:35:35.409+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_V3.third_task scheduled__2024-04-15T00:00:00+00:00 [running]> on host 5508ea9ebfaf
[2024-09-10T07:35:35.694+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='our_first_dag_V3' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2024-04-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-15T00:00:00+00:00'
[2024-09-10T07:35:35.695+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-10T07:35:35.732+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-10T07:35:35.735+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, this is the third task!']
[2024-09-10T07:35:35.752+0000] {subprocess.py:86} INFO - Output:
[2024-09-10T07:35:35.764+0000] {subprocess.py:93} INFO - hey, this is the third task!
[2024-09-10T07:35:35.765+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-10T07:35:35.844+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-10T07:35:35.846+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag_V3, task_id=third_task, run_id=scheduled__2024-04-15T00:00:00+00:00, execution_date=20240415T000000, start_date=20240910T073535, end_date=20240910T073535
[2024-09-10T07:35:35.956+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-10T07:35:35.990+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-10T07:35:36.019+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
